services_app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,272 @@
+                            android:layout_marginEnd="16dp"
+                            android:fontFamily="@font/montserrat"
+                            android:textColor="#474747 "
+                            android:textSize="18sp"
+                            app:layout_constraintBottom_toBottomOf="@+id/tvUserStateLabelMain"
+                            app:layout_constraintEnd_toEndOf="parent"
+                            app:layout_constraintStart_toEndOf="@+id/tvUserStateLabelMain"
+                            app:layout_constraintTop_toTopOf="@+id/tvUserStateLabelMain" />
+
+                    </androidx.constraintlayout.widget.ConstraintLayout>
+
+                </com.google.android.material.card.MaterialCardView>
+
+                <TextView
+                    android:id="@+id/tvUsernameLabelMain"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="24dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginBottom="8dp"
+                    android:fontFamily="@font/montserrat"
+                    android:text="@string/main_activity_username_label"
+                    android:textAlignment="viewEnd"
+                    android:textColor="#1A964E"
+                    android:textSize="18sp"
+                    android:textStyle="bold"
+                    app:layout_constraintBottom_toTopOf="@+id/tvLastSyncTimeLabelMain"
+                    app:layout_constraintEnd_toStartOf="@+id/tvUsernameMain"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/mcvMainDetails" />
+
+                <TextView
+                    android:id="@+id/tvUsernameMain"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="8dp"
+                    android:layout_marginEnd="24dp"
+                    android:fontFamily="@font/montserrat"
+                    android:textColor="#474747 "
+                    android:textSize="18sp"
+                    app:layout_constraintBottom_toBottomOf="@+id/tvUsernameLabelMain"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toEndOf="@+id/tvUsernameLabelMain"
+                    app:layout_constraintTop_toTopOf="@+id/tvUsernameLabelMain" />
+
+                <TextView
+                    android:id="@+id/tvLastSyncTimeLabelMain"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="24dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginBottom="8dp"
```

Comment:
Create dimens in dimen.xml, it will make this value consistent everywhere and errors can be avoided at later stage when we change this value (and forget to change at other places).

Reply:
Likewise for the colors. They should be defined in a separate xml file.